### DISCUSSION TITLE
Symlink creation stopped working in 18.0?

### DISCUSSION THREAD

**Discussion by s016aq** [Romanitho/Winget-AutoUpdate#555] [General]
Hello!
I'm in the process of updating our initial version in Intune to 1.19.1
I noticed that the symlink to the logfile is not being created during install any more (from version 1.18?)
Instead it's being created when the scheduled task is triggered (Winget-Upgrade.ps1)
But this does not happen for me.....
I get: **09:21:11 - Creating SymLink for log file (WAU-updates) in Intune Management Extension log folder** in updates.log
But it is not created!
If I run Winget-Upgrade.ps1 from Powershell_ise (as SYSTEM) it works. But not with the Scehduled task.
How can this be? Anyone else seeing this?

Kind Regards

**Comment by KnifMelti** [comment]
Confirmed.
Mark it as an issue, please!
